Sources/TensorFlow/LazyTensor/LazyTensorOperation.swift

```diff
@@ -0,0 +1,550 @@
+import CTensorFlow
+
+/// The `TF_Tensor *` type.
+typealias CTFTensor = OpaquePointer
+
+public class LazyTensor : _AnyTensorHandle {
```

Comment:
```suggestion
public class LazyTensor: _AnyTensorHandle {
```
The Google Swift Style Guide recommends no whitespaces before colons in conformance constraints.